packages/core/src/components/BrandConcierge/validateMessage.js

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+  anyOf,
+  arrayOf,
+  objectOf,
+  string,
+} from "../../utils/validation/index.js";
+
+export default ({ options }) => {
+  const brandConciergeEventValidator = anyOf([
+    objectOf({
+      message: string().required(),
```

Comment:
Ideally we'd like to make this command as close to sendEvent as possible. Is message needed long term or is it just a stop-gap measure?